clouddq/classes/dq_configs_cache.py

```diff
@@ -149,6 +150,25 @@ def get_row_filter_id(self, row_filter_id: str) -> dq_row_filter.DqRowFilter:
         )
         return row_filter
 
+    def get_reference_columns_id(
+        self, reference_columns_id: str
+    ) -> dq_reference_columns.DqReferenceColumns:
+        reference_columns_id = reference_columns_id.upper()
+        try:
+            reference_columns_record = self._cache_db["reference_columns"].get(
+                reference_columns_id
+            )
```

Comment:
if this returns a json string of a list, you need to parse the output before passing it into the class.